src/Panel/Redirect.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Kirby\Panel;
+
+use Exception;
+
+class Redirect extends Exception
```

Comment:
I feel like this class should be part of `Kirby\Exception`. But I'm not 100% sure.

Reply:
I was thinking about it as well, but we have this weird custom exception handling format in our exceptions and I wasn't sure if it fits in there well enough.

Reply:
We also have `ErrorPageException` there. But you are right, maybe it's too Panel specific.
We should rename the class to `RedirectException` though (common naming scheme for all kinds of exceptions).

Reply:
Idea: What if we move the exception to `Kirby\Exception\RedirectException` and also add support for this to our CMS router (like `ErrorPageException`)? Then it's actually generally useful.

Reply:
The custom exception handling code can actually be useful as we can use it to store the HTTP code of the redirect (301 or 302).